assets/js/base/utils/bhe-frontend.tsx

```diff
@@ -0,0 +1,160 @@
+/**
+ * External dependencies
+ */
+import { ReactElement } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { matcherFromSource, pickKeys } from './utils';
+import { hydrate } from './bhe-element';
+
+declare global {
+	interface Window {
+		blockTypes: Map< string, ReactElement >;
+	}
+}
+
+declare global {
+	// eslint-disable-next-line @typescript-eslint/no-namespace, @typescript-eslint/no-unused-vars
+	namespace JSX {
+		interface IntrinsicElements {
+			// eslint-disable-next-line @typescript-eslint/naming-convention
+			'gutenberg-inner-blocks': React.DetailedHTMLProps<
```

Comment:
We should probably also update the "nomenclature" here, see https://github.com/woocommerce/woocommerce-blocks-hydration-experiments/pull/21#issuecomment-1195598808 🙂 

(tl;dr `gutenberg` ➡️ `wp`, or `wp-block` in other contexts)

Reply:
I've taken the liberty of applying some nomenclature fixes in f7b465b. Feel free to revert if you don't like them! 😄

Reply:
Thanks @ockham ! I was going to do the same thing but you beat me to it :)